src/constants/storageKeys.ts

```diff
@@ -0,0 +1,4 @@
+export const STORAGE_KEYS = {
+  LOGIN_ID: 'loginId',
+  DARK_MODE: 'darkMode'
+  };
```

Comment:
you can maintain this in constants/constants file only, no need to maintain separate file for STORAGE_KEYS.
Unless there is specific reason to do